ansible/roles/artifactory/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Install Python dependencies
+  pip:
+    name: ['boto', 'boto3', 'botocore']
```

Comment:
Can we list these packages on separate lines please. That way any future changes will highlight on a line by line basis